src/centos/macros.tpl

```diff
@@ -27,7 +27,9 @@ python
 {% endmacro %}
 
 {% macro permissions_setup(spec) %}
-{% if spec.el_version == '7' %}
+{% if spec.version == '3.8' %}
```

Comment:
Can we make this `> 3.8` for future proofing? I'm not sure if it'll work though for e.g. `3.10`, might be a string comparison :/

Reply:
After the advice Petr gave me at the meeting today, I've reversed the condition so it now contains older versions of Python and we don't have to update it with every new Python version nor make it compare versions in the right way.